apps/src/blockly/addons/cdoSerializationHelpers.ts

```diff
@@ -414,8 +419,10 @@ export function convertFunctionsXmlToJson(functionsXml: string) {
   const parser = new DOMParser();
   const xml = parser.parseFromString(`<xml>${functionsXml}</xml>`, 'text/xml');
   const tempWorkspace = new Blockly.Workspace();
-  Blockly.Xml.domToBlockSpace(tempWorkspace, xml);
-  const proceduresState = Blockly.serialization.workspaces.save(tempWorkspace);
+  Blockly.Xml.domToBlockSpace(tempWorkspace, xml.children[0]);
```

Comment:
I needed to change this because we were passing a `Document` when we should have been passing an `Element`. This is safe to do because the parameter is parsed (in `cdoXml.ts` with `Array.from(xml.querySelectorAll('xml > block'));` which returns the same array for both.